benefits/oauth/views.py

```diff
@@ -27,6 +28,8 @@
 
 
 def login(request):
+    analytics.started_sign_in(request)
```

Comment:
Can we move this to after the (possible) `raise` - I don't think we should send an event if the app is somehow misconfigured.

Reply:
Sure, I'm ok with this. 

Would we want the same thing for `logout`? I would need to pull
```python
if not oauth_client:
   raise Exception("No OAuth client"
```
out of `_deauthorize_redirect` and to the top of `logout`. Then move the analytics line after the possible `raise`.

Reply:
Hmm pulling that out doesn't feel right? And `logout` still logs the user out of the the app, even if this part fails, right? (If the user `started` but didn't `finish` that might give us an indication that something was wrong in between)

I think no, but you go ahead and make the call whatever makes the most sense.

Reply:
Oh, I see the difference now. I agree with your reasoning, so I'll move it for login only.

Reply:
Nice, thank you!